target/linux/mpc85xx/p1020/config-default

```diff
@@ -44,12 +56,6 @@ CONFIG_RWSEM_SPIN_ON_OWNER=y
 CONFIG_SMP=y
 CONFIG_SPI_GPIO=y
 CONFIG_SWCONFIG=y
```

Comment:
CONFIG_SWCONFIG can also be dropped.

Reply:
I’ll have to look at this again. After this PR, the only device needing SWCONFIG is https://github.com/openwrt/openwrt/commit/97e4311fca73d064d17065e7844699aa777cb157

Reply:
Yep, WDR4900 conversion to DSA broke the Sophos RED 15w Rev.1.

That's 3 non working devices on this target. Panda, Sophos RED 15w Rev.1 , and WDR4900 because of wrong CONFIG for the PHY driver.